stripe/src/main/java/com/stripe/android/model/Stripe3ds2Fingerprint.kt

```diff
@@ -85,14 +86,36 @@ internal class Stripe3ds2Fingerprint private constructor(
         internal fun create(sdkData: StripeIntent.SdkData): Stripe3ds2Fingerprint {
             require(sdkData.is3ds2) { "Expected SdkData with type='stripe_3ds2_fingerprint'." }
 
-            return Stripe3ds2Fingerprint(
-                (sdkData.data[FIELD_THREE_D_SECURE_2_SOURCE] as String),
-                DirectoryServer.lookup((sdkData.data[FIELD_DIRECTORY_SERVER_NAME] as String)),
-                (sdkData.data[FIELD_SERVER_TRANSACTION_ID] as String),
-                DirectoryServerEncryption.create(
-                    (sdkData.data[FIELD_DIRECTORY_SERVER_ENCRYPTION] as Map<String, *>)
-                )
-            )
+            when (sdkData.data) {
+                is Either.Left -> {
+                    val (data: Map<String, *>) = sdkData.data
```

Comment:
remove parens?

Reply:
So maybe this is just being too clever but I was using here - https://kotlinlang.org/docs/reference/multi-declarations.html

Reply:
Removing this since it's just more complicated than necessary.